pandas/tests/groupby/test_missing.py

```diff
@@ -43,6 +43,17 @@ def test_ffill_missing_arguments():
         df.groupby("b").fillna()
 
 
+def test_ffill_with_string_column():
```

Comment:
rename to test_ffill_with_string_dtype

Reply:
I thought it made sense to rename to `test_fillna_with_string_dtype`, since I have now parameterized the test to test both `ffill` and `bfill`.